src/llm_utils.py

```diff
@@ -54,18 +54,13 @@ def analyze_code_changes(diff_content: str) -> List[Dict]:
         model="gemini-2.0-flash", contents=prompt
     )   
 
-    #print(response.text)
```

Comment:
This print statement should ideally be a logging statement for debugging purposes and should be removed in production code.

```suggestion
logging.debug(response.text)
```